app/models/book_rating.rb

```diff
@@ -4,5 +4,5 @@ class BookRating < ApplicationRecord
 
   validates :user_id, uniqueness: { scope: :book_id }
 
-  validates :rating_value, inclusion: { in: 0..10 }
+  validates :rating_value, inclusion: { in: 0..10 }, presence: true
```

Comment:
You have validation in Rails app, but have no validation in DB level, it means that it is theoretically possible to INSERT incorrect value to rating_value using SQL query from Postgrsql console. You should add database constraint to check this value

https://www.postgresql.org/docs/9.4/static/ddl-constraints.html